File: solutions/csharp/reverse-string/1/ReverseString.cs
using System;
using System.Text;

public static class ReverseString
{
    public static string Reverse(string input)
    {
        StringBuilder output = new StringBuilder(input);

        for (int i = 0; i < input.Length / 2; i++)
        {
            char temp = input[i];
            output[i] = input[input.Length - i - 1];
            output[input.Length - i - 1] = temp;
        }
        return output.ToString();
    }
}

File: solutions/java/reverse-string/1/src/main/java/ReverseString.java
class ReverseString {

    String reverse(String inputString) {
        StringBuilder outputString = new StringBuilder(inputString);

        for (int i = 0; i < inputString.length() / 2; i++) {
            char temp = inputString.charAt(i);
            outputString.setCharAt(i, inputString.charAt(inputString.length() - i - 1));
            outputString.setCharAt(outputString.length() - i - 1, temp);
        }

        return outputString.toString();
    }
  
}
